src/sensorpush_ble/parser.py

```diff
@@ -118,16 +173,19 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
             # and we need to wait for the next update.
             return
 
-        if data := _find_latest_data(changed_manufacturer_data):
-            device_type_id = 64 + (data[0] >> 2)
+        if data := _find_latest_data(changed_manufacturer_data, is_ht1):
+            device_type_id = 1 if is_ht1 else 64 + (data[0] >> 2)
             if known_device_type := SENSORPUSH_DEVICE_TYPES.get(device_type_id):
                 device_type = known_device_type
             result.update(decode_values(data, device_type_id))
 
-        if service_info.name.startswith("SensorPush "):
-            self.set_device_name(service_info.name[11:])
+        if is_ht1:
+            self.set_device_name(f"HT1 {service_info.address[12:].replace(':', '')}")
```

Comment:
Please use `short_address` here 
https://github.com/Bluetooth-Devices/bluetooth-data-tools/blob/e260e5b51d678131861320a49a06595c5742ed07/src/bluetooth_data_tools/__init__.py#L20

Reply:
Means bumping the project up to requring python 3.10, but I've done it